sdk/core/Azure.Core/tests/HttpPipelineFunctionalTests.cs

```diff
@@ -628,7 +628,7 @@ public async Task SendMultipartformData()
         }
 
         [Test]
-        public async Task HandlesRedirects()
+        public async Task HandlesRedirects([Values(true, false)] bool allowRedirects)
```

Comment:
Do we have tests of precedence rules of options vs. setting values on the message?

Reply:
added